tests/tests/tier0/bluechi-agent-resolve-fqdn/test_bluechi_agent_resolve_fqdn.py

```diff
@@ -29,7 +29,7 @@ def verify_resolving_fqdn(ctrl: BluechiControllerContainer, _: Dict[str, Bluechi
     local_node_cfg.manager_host = "localhost"
     ctrl.create_file(local_node_cfg.get_confd_dir(), local_node_cfg.file_name, local_node_cfg.serialize())
 
-    ctrl.exec_run('systemctl start bluechi-agent')
+    ctrl.systemctl_start_and_wait("bluechi-agent", 1)
```

Comment:
1 sec seems very fast to wait to me but maybe CI/CD these days have good machines.

Reply:
What `systemctl_start_and_wait` does is starts the service and calls `wait_for_unit_state_to_be()`. 
This param is additional sleep time. When running without valgrind it is not needed at all, but with valgrind things are a bit slower, so we need this additional timeout after service became `active`. This would also cover when running integration tests without timer, but maybe on slower infra.

I don't think that we should add more than that, since it is explicit sleep command, which makes all the tests run more time, and it becomes significant when the test count grows. From what I've seen, 1 second of additional sleep is more than.